recipes/aiopeewee/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "aiopeewee" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "03578a1ce28879f5e575c82fce8ead74a0e5ae0f7e2d2ac592f86207bf0be85e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+
+  run:
+    - python
+    - peewee >=2.9.2
+    - aiomysql >=0.0.9
+
+test:
+  imports:
+    - aiopeewee
+
+about:
+  home: https://github.com/kszucs/aiopeewee
+  summary: "Asyncio interface for Peewee ORM"
+  license: MIT
```

Comment:
I see this is your project: you should make a `LICENSE` file, put it in `MANIFEST.in`, and either make a new release with it or package it manually here until you do. (The MIT license requires redistributions to include a copy of the license text.)

Reply:
Done